charts/memgraph/templates/service.yaml

```diff
@@ -15,5 +15,12 @@ spec:
       targetPort: {{ .Values.service.targetPort}}
       protocol: {{ .Values.service.protocol }}
       name: bolt
-  selector:
-    {{- include "memgraph.selectorLabels" . | nindent 4 }}
+    - port: 7444
+      protocol: {{ .Values.service.protocol }}
+      name: websocket
+    - port: 9091
+      protocol: {{ .Values.service.protocol }}
+      name: http
+    
+  selector: 
+    {{- include "memgraph.selectorLabels" . | nindent 4 }}
```

Comment:
If you noticed we have a pre-commit hook on the repo and helm-linting should pass, try to make that work locally on your example. Note that some YAML formatting is ignored for charts since they are specific. Details in the precommit hook action.